src/index.ts

```diff
@@ -285,6 +291,10 @@ export async function build({
 		lambdaEnv.DENO_UNSTABLE = '1';
 	}
 
+	if(denoTsConfig) {
+		lambdaEnv.DENO_TSCONFIG = <string> env.DENO_TSCONFIG;
```

Comment:
```suggestion
	if (denoTsConfig) {
		lambdaEnv.DENO_TSCONFIG = denoTsConfig;
```

Reply:
Lambda is not assignable to type string | undefined, that's because env basically is an object { [prop: string]: string | undefined } and lambda's not. so there's that.

Reply:
Right, but the `if` check removes the possibility of it being undefined, so TypeScript is smart enough to know that it's a string.